src/ensembl/src/content/app/custom-download/containers/content/filter-accordion/FiltersAccordion.tsx

```diff
@@ -13,13 +13,30 @@ import {
 import styles from './FiltersAccordion.scss';
 
 import { getFiltersAccordionExpandedPanel } from './state/filterAccordionSelector';
-import { setFiltersAccordionExpandedPanel } from './state/filterAccordionActions';
+import {
+  setFiltersAccordionExpandedPanel,
+  resetSelectedFilters,
+  updateSelectedFilters
+} from './state/filterAccordionActions';
+
+import customDownloadStorageService from 'src/content/app/custom-download/services/custom-download-storage-service';
+import JSONValue from 'src/shared/types/JSON';
 
 import { Genes } from './sections';
+import ImageButton, {
+  ImageButtonStatus
+} from 'src/shared/image-button/ImageButton';
+import { ReactComponent as ResetIcon } from 'static/img/shared/trash.svg';
 
 type Props = StateProps & DispatchProps;
 
-const Filters = (props: Props) => {
+const FiltersAccordion = (props: Props) => {
+  useEffect(() => {
+    props.updateSelectedFilters(
+      customDownloadStorageService.getSelectedFilters()
```

Comment:
Should this not better be done via redux layer?

Reply:
That's done.